app/Http/Controllers/CsvController.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Support\Facades\Response;
+use Illuminate\Support\Facades\File;
+use App\Models\ImportMeta;
+use App\Models\Mismatch;
+use Carbon\Carbon;
+
+class CsvController extends Controller
+{
+    
+    public function get_csv()
+    {
+
+        $imports = ImportMeta::get();
+        $currentTime = Carbon::now()->format('Y-m-d');
+
+        $headers = array(
+            'Content-Type' => 'application/vnd.ms-excel; charset=utf-8'
```

Comment:
I think we should  use `text/csv` here to make this as interoperably and standardized as possible. See: https://stackoverflow.com/a/7076079

Reply:
yup i agree.